cli/install/install.go

```diff
@@ -434,20 +471,42 @@ func installTt(binDir string, installCtx InstallCtx, distfiles string) error {
 		}
 
 		if !versionFound {
-			return fmt.Errorf("%s version of tt doesn't exist", ttVersion)
+			commits, err_hashes := getTTCommits(installCtx.Local, distfiles)
+
+			if len(ttVersion) < 7 {
```

Comment:
Why do we check version length after getting commits list?

Reply:
fixed